docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -474,8 +490,11 @@ by the verifier plugin entry points defined in the `oe_verifier_t` structure:
     so the OE SDK framework can free it, and there is no need of a plugin entry
     point.
 - Verify evidence, optionally using a set of input endorsements and policies.
-  - Entry point `oe_result_t (*verify_evidence)(oe_verifier_t* context, const uint8_t* evidence_buffer, size_t evidence_buffer_size, const uint8_t* endorsements_buffer, size_t endorsements_buffer_size, const oe_policy_t* policies, size_t policies_size, oe_claim_t** claims, size_t* claims_length)`.
-  - This entry point has the same definition as in the OE SDK V0.9 release.
+  - Entry point `oe_result_t (*verify_evidence)(oe_verifier_t* context, const oe_uuid_t* format_id, const uint8_t* evidence_buffer, size_t evidence_buffer_size, const uint8_t* endorsements_buffer, size_t endorsements_buffer_size, const oe_policy_t* policies, size_t policies_size, oe_claim_t** claims, size_t* claims_length)`.
+  - The optional parameter `format_id` is added, to specify the
```

Comment:
Each verifier plugin implements this function so the 'format id' is already known (you use the format id to find this plugin). Why adding it?

Reply:
An single entry point can be invoked by multiple plugins, so its implementation needs to know which plugin calls it. The `format_id` serves that purpose.